src/modules/universal-discovery/services/universal.discovery.service.js

```diff
@@ -259,3 +283,121 @@ export const loadContentTypes = ({ token, siteaccess }, callback) => {
         .then(callback)
         .catch((error) => console.log('error:load:content:info', error));
 };
+
+/**
+ * Returns basic RequestInit object for Request
+ *
+ * @function getBasicRequestInit
+ * @param {Object} restInfo REST config hash containing: token and siteaccess properties
+ * @returns {RequestInit}
+ */
+const getBasicRequestInit = ({ token, siteaccess }) => {
+    return {
+        headers: {
+            'X-Siteaccess': siteaccess,
+            'X-CSRF-Token': token,
+        },
+        mode: 'cors',
+        credentials: 'same-origin',
+    };
+};
+
+/**
+ * Loads bookmarks
+ *
+ * @function loadBookmarks
+ * @param {Object} restInfo REST config hash containing: token and siteaccess properties
+ * @param {Function} callback
+ */
+export const loadBookmarks = (restInfo, callback) => {
+    const basicRequestInit = getBasicRequestInit(restInfo);
+
```

Comment:
not needed empty line

Reply:
I put this empty line on purpose. I think with it code looks nicer. 
I will remove it.